src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraAuthorizationException.scala

```diff
@@ -0,0 +1,16 @@
+
+package org.broadinstitute.dsde.agora.server.business
+
+import org.broadinstitute.dsde.agora.server.dataaccess.acls.AgoraPermissions
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+
+case class AgoraAuthorizationException(agoraPermissions: AgoraPermissions, agoraEntity: AgoraEntity) extends Exception {
+  override def getMessage: String = {
+    "Authorization exception attempting to exercise permission %d on entity %s.%s.%d. %s".format(
```

Comment:
I think string interpolation makes it easier to read what the string is meant to represent. I'll change it.